### PR TITLE
Creation any23 bundle

### PR DESCRIPTION
**Pull request by betoflowv93**
Hi , I'm a student of GSoC, currently I'm collaborating with the apache camel project. I need include this bundle for this task. Because, I'm working on karaf platform support for apache any23. Please , give it a look. 

thanks, Roberto.

### CODE REVIEW

File: any23-2.3/pom.xml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+            org.apache.any23.writer     
+        </servicemix.osgi.export.pkg>
+        <servicemix.osgi.import.pkg>
+            org.apache.any23.configuration,
+            org.apache.any23.source,
+            org.apache.any23.writer,
+            *
+        </servicemix.osgi.import.pkg>
+    </properties>
+      <!--
```
Comment (oscerd) [inline any23-2.3/pom.xml]: Is this commented stuff needed?

**Comment by oscerd** [comment]
Thanks

**Comment by jbonofre** [comment]
For the record, don't forget to update the main `pom.xml` in order to include the bundle in the next release cycle.

**Comment by jbonofre** [comment]
The parent pom should be in version 14, not 13, I will fix that at release time.

**Comment by jbonofre** [comment]
The bundle is not correct. If you want to use multiple line of export you have to use `servicemix.export`, not `servicemix.export.pkg`. I will fix that as well.

**Comment by oscerd** [comment]
Sorry for this. I didn't notice the missing entry in the pom.

**Pull request merged**